pkgs/development/python-modules/faker/default.nix

```diff
@@ -31,6 +31,8 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py --replace "pytest>=3.8.0,<3.9" "pytest"
+    # Remove pycache dirs that cause the tests to fail with Python 3.6
+    find tests -path '*/__pycache__*' -delete
```

Comment:
Just a heads up, this was resolved in master.

Reply:
Fixed